apps/communications/dialer/test/marionette/lib/views/activities/views.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+/* global module */
+
+var ActivitiesAccessor = require('./accessors');
+
+var appRoot = require('app-root-path');
+// TODO Change the path once requireFromApp becomes its own module
+var fromApp = require(appRoot +
+  '/shared/test/integration/require_from_app');
+
+function ActivitiesView(client) {
+  this.client = client;
+
+  this.accessors = new ActivitiesAccessor(client);
+
+}
+
+ActivitiesView.prototype = {
+
+
+  tapCreateContactOption: function() {
+      this.client.waitFor(function() {
+        return this.accessors.addNewContact.enabled();
+      }.bind(this));
+      var button = this.accessors.addNewContact;
+      button.tap();
+
+      var Contacts = fromApp('contacts').require('lib/contacts');
```

Comment:
Johan, I triyng to change that, but I am getting this error: cannot find module: shared/test/integration/../../../apps/communications/contacts/test/marionette/lib/communications/contacts
Seems like getAppClass('...') takes that path and it is not the correct path :( . If I change to only getAppClass('contacts') or empty, will also fail as they way it builds the path would not be valid for contacts unless we add another folder called communications between lib and contacts.js...
Not sure if there is another way to overcome this...